shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -172,6 +172,13 @@
     public static final String THREAD_POOL_ACTIVE_THREADS = PREFIX + "segmentstore.thread_pool.active_threads";  // Histogram
 
     // Metrics in Controller
+    // KeyValueTable request counts
+    public static final String CREATE_KVTABLE = PREFIX + "controller.kvtable.created";
```

Comment:
Please add comments indicating what type of metric this is. See other metric names defined in this class.

Reply:
I have added more comments to make the meaning explicit. But names are modeled based on those for STREAM metrics and so it should be self explanatory.